app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -252,6 +252,30 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
+  @Test
+  fun testExploration_clickAudioIconTwice_contentDescription_changesToDefault() {
+    setupAudioForFraction()
```

Comment:
Should be ``setUpAudioForFractionLesson`` ('setUp' being the main change). While not part of your PR, it'd be nice to fix this.

Reply:
Done.